### DISCUSSION TITLE
Properly Appoligize for such an idiot question pertaining to my SB F405 Wing FC and the latest stable iNAV.....

### DISCUSSION THREAD

**Discussion by coleon1977** [iNavFlight/inav#10111] [Q&A]
Quad Noob - Not Really
Wing Guy - Im Trying....
Kinda Possibly a Wing Noob..
No Reason to Cut Corners here....

Glad Im at this Mark Actually...

Anyway..
Wing's.....

3 months ago, I was gifted an Old School Partial set-up an older friend left for me. So Ya, I have that Behind this, which might explain my Need of Exact Certainty.. I Apologize if the questions I ask annoy you or enrage you enough to Blast my Naiveness  Regarding this aspect..
So like I mentioned, I was gifted an old school 72 MHz - Pre 1980 Cessna 172, with the Standard Balsa wood 6 ft wing span plane, filled with old school servo's that didn't work, a remote that had the wrong crystals in it. And I bought a FlySky FS-i6X TX along with the module to convert her from 72Mhs ----> the current 2,4G set-up.
And I did as I mentioned. Replaced, ordered, installed every component needed to function, in order to get this off the ground.

As Im Absolutely sure every Quad pilot, who has ever been down this road, has done EVERYTHING I've done to ensure this get's off the ground. 
Long Story Short - The plane is 110% ready for its maiden flight - And Im just being a Bitch, finding every excuse not to fly this thing bc I Absolutely do NOT want to Fuck this up. 
There's been 34 occasions I've ventured out to the appropriate sized field in hopes of conquering this task, and each time there has been an obsticle present that due to my age and experienced I've learned not to ush the envelope and ultimately doing something that would ruin or break it, resulting in a much longer delay in achieving my goal....

FF a couple weeks,  I purchased a ZOHD Dart XL, and began to buy and assemble in the same fashion as I do on all my build's. 

And at this very moment, I have bought, constructed, embraced, Enforced and strengthened those weak shit ass parts, for strong, functional 3D Printed parts, helping the over all longevity of this build, and at the moment,  Im sitting here wanting to make sure Im going about doing a CORRECT Structured back-up the correct way, and Im Asking if my approach is the correct way to go about this, or am I not seeing some dumb shit I should be....

I have to stop for a minute and Apologize....
Im sure theres like 1 or 2 people who feel like I do, and theres like 100 people who happened to get bored reading what I wrote 30 seconds after I started. Im just wanting someone to help me understand the steps in which I should take to ensure I'm doing a correct back-up, and moving forward doing things in the CORRECT fashion and order, ultimately being able to enjoy what all of us are after...

Sorry if I've ANNOYED Anyone with my ......

Whatever Questions...

**Comment by sensei-hacker** [comment]
Is your question "how do I make sure I don't completely crash in the first five seconds on my first flight?" I'm not sure what the question is.

Here are three things you can do to make sure you don't destroy your plane in the first few seconds:

Practice in a simulator.

Do the "high five" test (see YouTube for videos showing how).

Find an area with long wild grass, a hay field, or other area where a hard landing won't destroy the plane.  Throw it, motor off, over the long grass. Mostly straight ahead, just slightly up. Watch as it glides into the long grass. Adjust as needed to get a nice glide.  Once your plane glides nicely, adding power will make it keep going.

One other thing - you mentioned strengthening areas and 3D printing parts. Keep in mind weight is VERY important on an aircraft. If you added a new part that makes it stronger, that also makes it heavier. Similarly for replacing a part with a heavier part. A piece of paper or a cotton ball is weak and light, and won't break if you drop it from a height. A phone is much stronger and heavier and will break into many pieces if dropped from the third floor.  Be careful of adding weight.

Any weight you add becomes a hammer that effectively strikes whatever it is attached to during a hard landing.

**Reply by coleon1977** [reply]
Sensei Hacker ~ Thanks for your feedback and for taking the time to read & even Respond to my long-ass ridiculous rant into my current and past experiences / builds…

I guess one of the things I’m trying to understand is - In BetaFlight when connecting to any of my quads, up top I have options I can choose to be able to connect using the right usb ports, baud etc.

Mind you, This is my 1st go-around with iNAV.
But when I attempted to connect my SB F405 FC, those options weren’t present nor available for me to use.

You’ll have to forgive me as I’m not at home right now in front of my computer. But the only option I had looked to be only being able to connect via Bluetooth, which is a newer feature for Speedy Bee and is very convenient, but I wanted to go about this on my computer for the first time so I can get a broader picture on  Options when configuring .

So I was able to connect to the 405 via Bluetooth on my Dell laptop, but I was unable to open up any of the tabs on the side to do any configuring 

I will attempt to do this shortly and post the error messages that I received when trying to do that. 

So then, on my second attempt, I used a USB to USB cable, and I was able to connect to the 405 flight controller. But I was given the same error message and unable to utilize any of the tabs on the side to configure anything. 

I think I posted another question out there regarding this, but at the moment I am not able to search for it to me responding to your message..lol

I know that I do have the ability to change the modes in the flight controller from Bluetooth to USB I believe and then I think there’s two more as well. I think one says STL or something like that.

And when I went online to try and reference those options, I ran out of time, trying to figure out what they meant, and what I should be selecting versus choosing random options, hoping to be able to connect

One of the first things that I know, I need to do what I was initially trying to accomplish when connecting for the first time was to do a back up of the software that came on the F4 05 flight controller.

I have learned my lesson the hard way on a couple of occasions and I definitely do not want to do that with this bill due to the time and effort that I’ve put into this thus far

So essentially, I think I need to figure out how I need to get in and being able to use the software properly connected to controller.

And then I need to figure out how to do a backup of the original files that came on the Speedy Bee F4. 05 flight controller.

If you or anyone can assist me in that general area, I’d greatly appreciate that!!!!!


I Sooo just wanna configure this and get up in the Air… I’ve been looking at my completed ZOHD Dart-XL for a couple weeks now, and me, currently not being able to connect the dots - Is ABSOLUTELY Driving Me Bat Shit Crazy……

I just need to do an Appropriate Through Back-Up of the stock bin files..

And figure out what the options are in the drop down when wanting to connect in ways other than BT.using different ways to connect


I’ll post more specifics momentarily….

Thanks to ALL taking the time to read all of this

-Cole

**Comment by sensei-hacker** [comment]
Yeah a USB data cable is the most common way to connect. Note there are also USB *charging* cables which won't work - they only have the power ones and not the data wires. That should look a lot like Betaflight.

When you get a chance, let us know what the error message says. Also what version of Configurator you're using.
